src/hooks/useDragAndDrop.ts

```diff
@@ -23,9 +25,17 @@ type DragAndDropOptions = {
 /**
  * @param dropZone – ref to the dropZone component
  */
-export default function useDragAndDrop({dropZone, onDrop = () => {}, shouldAllowDrop = true, isDisabled = false, shouldAcceptDrop = () => true}: DragAndDropParams): DragAndDropOptions {
+export default function useDragAndDrop({
+    dropZone,
+    onDrop = () => {},
```

Comment:
Do you think it would be easy to add `onDragEnter`, symmetrical to `onDrop`?

I'm wondering if this would allow us to reduce the boilerplate in...

```js
        if (prevIsDraggingOver === isDraggingOver) {
            return;
        }
        if (isDraggingOver) {
        
        // ...
```

Reply:
Oh, `Suggestions` uses `useContext(DragAndDropContext);`... I'm still not convinced by it. I'll think about this.

Reply:
For now let's consider this thread "on hold"

Reply:
Let's do nothing here. We have to merge this PR some day.